tests/components/elgato/test_config_flow.py

```diff
@@ -27,12 +28,12 @@ async def test_full_user_flow_implementation(
     await hass.config_entries.flow.async_init(
         DOMAIN,
         context={CONF_SOURCE: SOURCE_ZEROCONF},
-        data={
-            "host": "127.0.0.1",
-            "hostname": "example.local.",
-            "port": 9123,
-            "properties": {},
-        },
+        data=zeroconf.HaServiceInfo(
```

Comment:
I find it super awkward this class is prefixed with `Ha`...

Reply:
It was added in https://github.com/home-assistant/core/pull/48450
Do you want to rename it?
I think it's only used in tests (and inside zeroconf/init.py)

Reply:
Cc @KapJI, @bdraco

Reply:
I'm fine with renaming it ZeroconfServiceInfo